step-automation-packages/step-automation-packages-manager/src/main/java/step/automation/packages/AutomationPackageManager.java

```diff
@@ -504,20 +518,24 @@ protected void persistStagedEntities(Staging staging,
             objectEnricher.accept(execTasksParameter.getExecutionsParameters());
```

Comment:
@iegorov777 this should be moved to the onCreate of the schedule hook, we should not have any direct reference to executionTasksParameter here

Reply:
@david-stephan 
Agree

Reply:
@david-stephan 
Fixed